hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -418,59 +418,62 @@
     {
       name: chip_jtag_csr_rw
       desc: '''
-            Verify accessibility of CSRs as indicated in the RAL specification.
+            Verify accessibility of all the CSRs in the chip through jtag and also test RV_DM SBA
+            functionality
 
             - Shuffle the list of CSRs first to remove the effect of ordering.
             - Write all CSRs via JTAG interface with a random value.
             - Shuffle the list of CSRs yet again.
             - Read all CSRs back and check their values for correctness while adhering to the CSR's
               access policies.
+            - When the CSRs is in other module, the access goes through RV_DM SBA interface into
+              the xbar.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_rv_dm_cpu_debug_mem
-      desc: '''Verify access to the debug mem from the CPU.
-            '''
-      milestone: V2
-      tests: []
-    }
-    {
-      name: chip_rv_dm_jtag_debug_mem
-      desc: '''Verify access to the debug mem from the external JTAG interface.
+      name: chip_rv_dm_cpu_debug_mem_not_accessable
+      desc: '''Verify that the debug mem can't be accessed from the CPU while not in the debug mode.
```

Comment:
Missing stimulus and checks.

Reply:
We'd want a test that attempts to read, write and execute from the debug memory, seeing a synchronous exception on all of these accesses.

Is blocking off the debug memory something that's happening external to Ibex? As I don't think Ibex does this internally

Reply:
it's not clear to me. Since I have a TODO here, perhaps let's figure it out later.